thirdparty/openjdk-cacerts/zimbra-openjdk-cacerts/rpm/SPECS/openjdk-cacerts.spec

```diff
@@ -13,24 +13,37 @@ AutoReqProv:        no
 %description
 CA certs keystore for use with OpenJDK
 
+%changelog
+* Thu Feb 11 2016  Zimbra Packaging Services <packaging-devel@zimbra.com> - 1.0.5-ITERATIONZAPPEND
+- Relocate cacerts to OZCE/java to avoid conflicts with the OpenJDK package
+* Fri Dec 14 2015  Zimbra Packaging Services <packaging-devel@zimbra.com> - 1.0.4-ITERATIONZAPPEND
+- Support running zmcertmgr as user zimbra instead of root
+* Fri Dec 11 2015  Zimbra Packaging Services <packaging-devel@zimbra.com> - 1.0.3-ITERATIONZAPPEND
+- Enhance upgrade check
+* Mon Dec 07 2015  Zimbra Packaging Services <packaging-devel@zimbra.com> - 1.0.2-ITERATIONZAPPEND
+- Ensure that on upgrades of the zimbra-openjdk-cacerts package, that the keystore is updated
+- Ensure that on upgrades of the zimbra-openjdk-cacerts package, that the old keystore is backed up
+* Wed Dec 02 2015  Zimbra Packaging Services <packaging-devel@zimbra.com> - 1.0.1-ITERATIONZAPPEND
+- Updated to include full CA Certs from Mozilla CA list
+
 %install
-mkdir -p ${RPM_BUILD_ROOT}/OZCL/jvm/java/jre/lib/security/
-cp ../../cacerts ${RPM_BUILD_ROOT}/OZCL/jvm/java/jre/lib/security/cacerts
+mkdir -p ${RPM_BUILD_ROOT}/OZCE/java
+cp ../../cacerts ${RPM_BUILD_ROOT}/OZCE/java/cacerts
```

Comment:
Same as earlier comment: should we chain these two together via && so we only do step 2 (cp) if mkdir is successful?